babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -236,6 +236,35 @@ SqlCreateAttribute CreateTableAttributeJournalTable() :
     { return new SqlCreateAttributeJournalTable(id, getPos()); }
 }
 
+SqlCreateAttribute CreateTableAttributeDataBlockSize() :
+{
+    DataBlockModifier modifier = null;
+    DataBlockUnitSize unitSize;
+    SqlLiteral dataBlockSize = null;
+}
+{
+    (
+        (
+            ( <MINIMUM> | <MIN> ) { modifier = DataBlockModifier.MINIMUM; }
+        |
+            ( <MAXIMUM> | <MAX> ) { modifier = DataBlockModifier.MAXIMUM; }
+        |
+            <DEFAULT_> { modifier = DataBlockModifier.DEFAULT; }
+        )
+        <DATABLOCKSIZE> { unitSize = DataBlockUnitSize.UNSPECIFIED; }
+    |
+        <DATABLOCKSIZE> <EQ> dataBlockSize = UnsignedNumericLiteral()
+        (
+            <BYTES> { unitSize = DataBlockUnitSize.BYTES; }
+        |
+            ( <KILOBYTES> | <KBYTES> ) { unitSize = DataBlockUnitSize.KILOBYTES; }
+        |
+            { unitSize = DataBlockUnitSize.UNSPECIFIED; }
```

Comment:
Should this be DataBlocUnitSize.BYTES? I saw in the DataBlockUnitSize enum definition that the comment for DataBlockUnitSize.UNSPECIFICIED says the default size is BYTES. If so, wouldn't it make sense to have BYTES be the default if the user doesn't specify a unit?

Reply:
That's a good point. If a user doesn't assign a unit size we could just default to BYTES, and appending BYTES when unparsing would just give the user an equivalent statement